src/messages/messagesEN.ts

```diff
@@ -97,6 +97,55 @@ const messages = {
       userUpdated: "User updated",
     },
     goToMyNdla: "Go to My NDLA",
+    learningpath: {
+      title: "My Learning Paths",
+      description:
+        "Here you can create your own learning paths and share them with your students. The learning paths can include articles from NDLA, links to other resources, and short texts you create yourself.",
+      created: "Created: {{ created }}",
+      shared: "Shared: {{ shared }}",
+      noPath: "It seems you have no learning paths created, click the `Add` button to start creating one!",
```

Comment:
Synes denne leser litt rart. Er det en oversettelse vi har fått fra NDLA, eller kan vi skrive den om litt?

Reply:
"You haven't created any learning paths yet. Click the `Add` button to start creating one"